src/components/Modals/FiatRamps/components/AssetSearch/AssetList.tsx

```diff
@@ -1,30 +1,28 @@
 import type { ListProps } from '@chakra-ui/react'
+import type { AssetId } from '@shapeshiftoss/caip'
 import { FixedSizeList } from 'react-window'
 import { Text } from 'components/Text'
 
-import type { FiatRampAction, FiatRampAsset } from '../../FiatRampsCommon'
+import type { FiatRampAction } from '../../FiatRampsCommon'
 import { AssetRow } from './AssetRow'
 
 type AssetListProps = {
-  handleClick: (asset: FiatRampAsset) => void
-  assets: FiatRampAsset[]
-  type: FiatRampAction
+  handleClick: (assetId: AssetId) => void
+  assetIds: AssetId[]
+  action: FiatRampAction
 } & ListProps
 
-export const AssetList = ({ assets, type, handleClick }: AssetListProps) => {
-  if (!assets.length) return <Text translation='common.noResultsFound' />
+export const AssetList: React.FC<AssetListProps> = props => {
+  const { assetIds } = props
```

Comment:
Something something strict memo, technically speaking react will shallow compare `assetIds` so we're totally good, but getting a new reference every render is something we want to avoid for strict memoization purposes